lib/libzfs/libzfs_import.c

```diff
@@ -846,12 +865,13 @@ label_offset(uint64_t size, int l)
  * describing the configuration, if there is one.
  */
 int
-zpool_read_label(int fd, nvlist_t **config)
+zpool_read_label(int fd, nvlist_t **config, int *labels)
```

Comment:
If the `labels` parameter was instead named something like `num_labels`, its use would be more clear and the confusion on line 927 would have been less likely.  Also, its use should be document in the function comment above.